src/core/database/storage/SettingsModelStorage.ts

```diff
@@ -84,6 +84,10 @@ export class SettingsModelStorage extends VersionedObjectStorage<Record<string,
                     description: 'Reset for fees update.',
                     migrate: () => undefined,
                 },
+                {
```

Comment:
- We need reset storage to remove explorer link from there?
- description is wrong